src/cppnv/EnvPair.cc

```diff
@@ -0,0 +1 @@
+#include "EnvPair.h"
```

Comment:
This file seems unnecessary

Reply:
EnvPair is the container class for the key/value. I could use a tuple but I prefer clearer semantics. Can you make your request clearer?

Reply:
I think what James means is: why would we want a `.cc` file that is empty other than one include?

It is also very unconventional (and likely undesirable) to create this many small C++ header and source files. As suggested elsewhere, it's probably best to use a single header and a single source file for this feature.

Reply:
As per someone else's request I've merged everything into two files in  node_dotenv.xx